taqueria-plugin-ligo/common.ts

```diff
@@ -18,7 +19,7 @@ export type IntersectionOpts = LigoOpts & CompileOpts & TestOpts;
 type UnionOpts = LigoOpts | CompileOpts | TestOpts;
 
 // Points to the latest stable version. Needs to update this as part of our release process
-export const LIGO_DOCKER_IMAGE = 'ligolang/ligo:0.55.0';
+export const LIGO_DOCKER_IMAGE = getDockerImage('ligolang/ligo:0.55.0', 'TAQ_LIGO_IMAGE');
```

Comment:
It is easier to reason and reduces cognitive overhead to only assign literals to global constants